Shelf.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Refrigerator_EX
+{
+    class Shelf
+    {
+        private static int nextID = 1;
+
+        private int ID;
+        private int floorNumber;
+        private int shelfSpace;
+        private List<Item> items;
+
+       
+        public int Id
+        { 
+            get{return ID; } 
+            set {; } 
+        }
+        public int FloorNumber
+        {
+            get { return floorNumber; } 
+            set {if (value > 0)
+                    floorNumber = value;
+                else
+                    Console.WriteLine("floorNumber must be positive");
+            }
+        }
+        public int ShelfSpace 
+        {
+            get { return shelfSpace; } 
+            set {
+                if (value > 0)
+                    shelfSpace = value;
+                else
+                    Console.WriteLine("shelfSpace must be positive");
+            }
+        }
+        public List<Item> Items 
+        {
+            get { return items; } 
```

Comment:
When you let external classes access your list you are destroying encapsulation of the class implementation.